### ISSUE TITLE
Potion Charge Bug

### ISSUE THREAD

**Issue by Raveglitch** [MelvorIdle/melvoridle.github.io#259]
**Describe the bug**
When using Generous Cook Potion in offline mode it consumes the potion as a 1 charge potion instead of the intended number of charges the potion should have.

**To Reproduce**
1. Go to Cooking.
2. Select fish of choice to cook and activate Generous Cook Potion.
3. Enable potion Auto-Reuse as active.
4. Close browser to go into offline mode.
5. Open browser to view number of potion used equivalent to number of fish cooked.

**Expected behavior**
The potions will be used up disregarding the actual charges each potion has.

**Screenshots**
![unknown](https://user-images.githubusercontent.com/66107232/83225872-90a08900-a135-11ea-9bb8-bf31c51cb313.png)

**Browser**
Chrome Version 83.0.4103.61 (Official Build) (64-bit)

**Console output**
Please open the console by pressing F12 and copy it here.

**Are you using any scripts?**
None

**Comment by MelvorIdle** [comment]
The wording states it used "8 Potion charges". Can you confirm if the bug actually occurs like you stated?

**Comment by Raveglitch** [comment]
Yes, i have posted a screen shot of it.

**Comment by Raveglitch** [comment]
Sorry, uhhhh here is the screenshots. The first time the bug occurred I did lose 200+ pots. 
The second time I guess I cannot confirm. 
And this is the third time I just tested and it seems to be just a notification bug.

![image](https://user-images.githubusercontent.com/66107232/83727360-c5ab5080-a5f9-11ea-883f-2dfa1ed7fcc6.png)
![image](https://user-images.githubusercontent.com/66107232/83727410-deb40180-a5f9-11ea-9dac-891e1b0fbf30.png)
![image](https://user-images.githubusercontent.com/66107232/83727463-f2f7fe80-a5f9-11ea-9703-a2f8c1ae5b62.png)
